yocto/shortlog.sh

```diff
@@ -26,14 +26,16 @@
 MANIFEST=$1
 YOCTO_BUILD_DIR="$2"
 SOURCE_CODE_ROOT="$3"
+BRANCH="$4"
+BRANCH=${BRANCH:-master}
```

Comment:
Can you change this to `${4:-master}` please?

Reply:
Sure, Done!